README.md

```diff
@@ -126,6 +132,80 @@ Where `tokenProvider` is a function that returns a fresh access token or an erro
 actually trigger the retrieval of a token, this happens when the first statment is issued and a connection
 is created.
 
+
+### Always Encrypted support (preview)
+
+`go-mssql` supports a client-side decryption of the column encrypted values for those databases
+that are using the [Always Encrypted](https://docs.microsoft.com/en-us/sql/relational-databases/security/encryption/always-encrypted-database-engine?view=sql-server-ver15)
+feature.  
+  
+To start using the feature, you have to use the following parameters in your DSN:
+
+* `columnEncryption=true`
+* `keyStoreAuthentication=pfx` - Only `pfx` is supported at the moment
+* `keyStoreLocation=/path/to/your/keystore.pfx` - The location of the key store file (e.g: `./resources/test/always-encrypted/ae-1.pfx`), used when `columnEncryption` is enabled
+* `keyStoreSecret=secret` - The password of your keystore (`keyStoreLocation`)
+
+#### Usage
+
+Using the Always Encrypted feature should be transparent in the driver:
+```go
+query := url.Values{}
+query.Add("database", "dbname")
+query.Add("columnEncryption", "true")
+query.Add("keyStoreAuthentication", "pfx")
+query.Add("keyStoreLocation", "./resources/test/always-encrypted/ae-1.pfx")
+query.Add("keyStoreSecret", "password")
+
+
+hostname := "172.20.0.2"
+port:= 1433
+
+u := &url.URL{
+    Scheme:   "sqlserver",
+    User:     url.UserPassword("sa", "superSecurePassword_"),
+    Host:     fmt.Sprintf("%s:%d", hostname, port),
+    RawQuery: query.Encode(),
+}
+
+db, err := sql.Open("sqlserver", u.String())
+if err != nil {
+    logrus.Fatalf("unable to open db: %v", err)
+}
+rows, err := db.Query("SELECT id, ssn FROM [dbo].[cid]")
+if err != nil {
+    logrus.Fatalf("unable to perform query: %v", err)
+}
+
+for ; rows.Next(); {
```

Comment:
In Go, there are three for loop variants: 0 argument (loop forever), 1 argument (test condition), and 3 argument.

You want the 1 argument version here: `for rows.Next() {`